extensions/sql-migration/src/main.ts

```diff
@@ -37,6 +42,29 @@ class SQLMigration {
 			vscode.commands.registerCommand('sqlmigration.testDialog', async () => {
 				let dialog = new AssessmentResultsDialog('ownerUri', undefined!, 'Assessment Dialog');
 				await dialog.openDialog();
+			}),
+
+			vscode.commands.registerCommand('sqlmigration.openNotebooks', async () => {
+				const input = vscode.window.createQuickPick<MigrationNotebookInfo>();
+				input.placeholder = loc.NOTEBOOK_QUICK_PICK_PLACEHOLDER;
+
+				input.items = NotebookPathHelper.getAllMigrationNotebooks();
+
+				input.onDidAccept(async (e) => {
+					const selectedNotebook = input.selectedItems[0];
```

Comment:
Make sure you're handling the case where they exit out of the dialog correctly (item will be undefined)